### PR TITLE
Extract parametric tracer version

### PR DESCRIPTION
**Pull request by robertomonteromiguel**
## Description

We need to know the version of the tracer under test before running the tests, to make the @missing_feature annotations work.

## Motivation

In python need to ignore tests for specific versions of tracer

## Workflow

1. ⚠️⚠️ Create your PR as draft
2. Follow the style guidelines of this project (See [how to easily lint the code](https://github.com/DataDog/system-tests/blob/main/docs/edit/lint.md))
3. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
4. Mark it as ready for review

Once your PR is reviewed, you can merge it! :heart:

## Reviewer checklist

* [ ] Check what scenarios are modified. If needed, add the relevant label (`run-parametric-scenario`, `run-profiling-scenario`...). If this PR modifies any system-tests internal, then add the `run-all-scenarios` label ([more info](https://github.com/DataDog/system-tests/blob/main/docs/CI/labels.md)). 
* [ ] CI is green
   * [ ] If not, failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] if any of `build-some-image` label is present
  1. is the image labl have been updated ? 
  2. just before merging, locally build and push the image to hub.docker.com
* [ ] if a scenario is added (or removed), add (or remove) it in system-test-dasboard nightly

### CODE REVIEW

**Review by bantonsson — APPROVED** [review]
The common and Java parts look ok to me.

**Review by marcotc — APPROVED** [review]
:ruby: approved

**Review by zacharycmontoya — APPROVED** [review]
LGTM for .NET

File: utils/build/docker/dotnet/parametric/ddtracer_version.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+FROM alpine/xml
+WORKDIR /client
+
+COPY ./utils/build/docker/dotnet/parametric/ApmTestClient.csproj .
+COPY ./utils/build/docker/dotnet/parametric/ddtracer_version.sh .
+RUN sh ddtracer_version.sh
```
Comment (cbeauchesne) [inline utils/build/docker/dotnet/parametric/ddtracer_version.Dockerfile:7]: As the sh file is mostly a one liner, you can shave it : 

```suggestion
RUN echo $(xmllint --xpath 'string(//Project/ItemGroup/PackageReference[@Include="Datadog.Trace.Bundle"]/@Version)' ApmTestClient.csproj) > SYSTEM_TESTS_LIBRARY_VERSION

```

**Review by cbeauchesne — CHANGES_REQUESTED** [review]

**Pull request merged**